lessons/08-html-site/index.html

```diff
@@ -0,0 +1,26 @@
+<html>
+    <title>Home|Jaynen Sukhadia</title>
+    <header>
+        <h1>
+            <img src="https://images-na.ssl-images-amazon.com/images/M/MV5BMTAyNTAzMTA4OTJeQTJeQWpwZ15BbWU3MDA4NDI2Njk@._V1_UX172_CR0,0,172,256_AL_.jpg"; width="50px"; height="50px"; alt="My image">
+        </h1>
+        <h2>Jaynen's Portfolio</h2>
+    <nav>
+        <a href="index.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="contact.html">Contact</a>
+    </nav>
+    </header>
+    <hr>
+     
+    <main>
+        <P style="font-size: xx-large">Home</p>
+        <p>Welcome to my portfolio. Click "About" to learn more about me.Click "Contact" to reach me directly.</p>
+        <p>You can use this form to search Google.</p>
+    </main>
+    <form action="https://google.ca/search"; method="GET">
+    <p>Search: <input type="text"; name="q">
+    <button>Search Google</button></p>
```

Comment:
Instead of placing this in a `<p>`, it would make more sense semantically to place it in a `<div>` because we aren't writing a paragraph

Reply:
ok. Corrected in new lessons.